src/Resources/config/services.yml

```diff
@@ -5,13 +5,17 @@ imports:
     - { resource: "@BitBagSyliusInvoicingPlugin/Resources/config/services/resolver.yml" }
 
 services:
+    bitbag_sylius_invoicing_plugin.file_generator.invoice_filename:
+        class: BitBag\SyliusInvoicingPlugin\FileGenerator\InvoicePdfFilenameGenerator
+
     bitbag_sylius_invoicing_plugin.file_generator.invoice_file:
         class: BitBag\SyliusInvoicingPlugin\FileGenerator\InvoicePdfFileGenerator
         arguments:
             - "@knp_snappy.pdf"
             - "@templating"
             - "@bitbag_sylius_invoicing_plugin.resolver.company_data"
             - "%invoices_root_dir%"
+            - "@bitbag_sylius_invoicing_plugin.file_generator.invoice_filename"
```

Comment:
In our convention, we put the object types above the default types, so if possible, interfaces over strings, arrays, etc. Could you please change this order remembering the right order in __construct call as well as properties order in `InvoicePdfFileGenerator`? 🙂